uaclient/snap.py

```diff
@@ -121,12 +121,12 @@ def get_installed_snaps() -> List[SnapPackage]:
 
 
 def install_snapd():
-    event.info(messages.APT_UPDATING_LISTS)
+    event.info(messages.APT_UPDATING_LIST.format("main"))
     try:
-        apt.run_apt_update_command()
+        apt.update_sources_list("/etc/apt/sources.list")
```

Comment:
This could go wrong if someone has a snapd candidate from another repository (or if they don't really maintain the ubuntu repos in sources.list but I think that is too odd to support). However, if they have a different candidate, it would be even weirder that it would be not installed. So I think although we can't guarantee this update will help, it's fine for  99.9% of the cases.

Reply:
I lean more towards keeping the 99% safeness and favor boot time. But, if you think it is more appropriate to be 100% and do a full update while enabling this service, please let me know and I will revert it.

What do @lucasmoura and @orndorffgrant think?

Reply:
nah let's keep this as-is: this was more of a point-of-attention comment than a request for change. This is good.